app/views/questions/show.html.erb

```diff
@@ -2,6 +2,11 @@ Title: <%= @question.title %><br><br>
 <img src="<%= @question.image %>"/> <br><br>
 Question: <%= @question.content %><br><br>
 
+Tags:
+  <% @tags.each do |tag| %>
+    <%= tag.title %>
+  <% end %><br><br>
```

Comment:
DemBoyz want to know...
Are you referring to the indentation? DemBoyz are not sure about what you mean.